modules/gdscript/gdscript.cpp

```diff
@@ -494,20 +511,8 @@ bool GDScript::_update_exports() {
 			changed = true;
 		}
 	}
-
-	if (placeholders.size()) { //hm :(
-
-		// update placeholders if any
-		Map<StringName, Variant> values;
-		List<PropertyInfo> propnames;
-		_update_exports_values(values, propnames);
-
-		for (Set<PlaceHolderScriptInstance *>::Element *E = placeholders.front(); E; E = E->next()) {
-			E->get()->set_build_failed(false);
-			E->get()->update(propnames, values);
-		}
-	}
-
+	
```

Comment:
I'm not sure why `clang-format` is still failing here in CI tests, as there doesn't seem to be any trailing tab anymore.